modules/ROOT/pages/dataweave-cookbook-extract-data.adoc

```diff
@@ -133,35 +133,44 @@ variables, for example) of one of the supported DataWeave value types.
 | Data to extract | Syntax
 
 | *Payload*
-| `#[payload]` or `payload`
+a| `payload`, for example: `payload.name`
+
+If the `payload` is `{"name" : "somebody"}`, `payload.name` returns `"somebody"`.
+
+For more on the Mule payload, see
```

Comment:
For more information... I see now that For more on xxx is used in multiple places. Is this a style requirement?  It sounds funny to me to not have "information on" after "For more..."